lib/src/sources/dmi_source.cc

```diff
@@ -82,7 +95,7 @@ namespace whereami { namespace sources {
         if (dmidecode.empty()) {
             LOG_DEBUG("dmidecode executable not found");
         } else {
-            int dmi_type {-1};
+            int dmi_type{-1};
```

Comment:
I just wasn't really aware of what the standard way to do it was - I will adjust to include the space!